demisto_sdk/commands/common/tests/tools_test.py

```diff
@@ -2488,3 +2489,10 @@ def test_find_type_by_path(path: Path, expected_type: Optional[FileType]):
 )
 def test_field_to_cliname(value: str, expected: str):
     assert field_to_cli_name(value) == expected
+
+
+def test_get_core_packs():
+    mp_to_core_packs = get_marketplace_to_core_packs()
```

Comment:
We have additional tests that require an internet connection. I prefer to test with the real files, and IMO if something is broken, it's actually good that it will fail and get fixed quickly (we can force merge if needed), these are important files.

Reply:
this is prone to connectivity issues, and will prevent _all_ SDK builds :\ I'm not too happy to approve it